fapolicy_analyzer/ui/main_window.py

```diff
@@ -431,8 +431,11 @@ def _refresh_toolbar(self):
         self.__toolbar.refresh_buttons_sensitivity()
 
     def on_deployChanges_clicked(self, *args):
+        unsaved_rule = False
+        if isinstance(self.__page, RulesAdminPage):
+            unsaved_rule = self.__page.get_is_dirty()
```

Comment:
I'd rather see this work done in the `DeployChangesetsOp` class instead of here. That class subscribes to the system feature so it should be able to get the information it needs to determine if the rules are dirty or not.